modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -0,0 +1,86 @@
+* Your Git repository maintainer keeps the `devfile.yaml` or `.devfile.yaml` file in the root directory of the Git repository. (For alternative file names and file paths, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].)
++
+IMPORTANT: Although the devfile enables full functionality of a workspace, you can also start a new workspace by supplying the URL of a Git repository without a devfile. This results in a new workspace with a clone of the Git repository and with the defaults such as the Che-Theia IDE and the Universal Developer Image.
+//provide xrefs to a concept module about the Universal Developer Image and, if applicable, devfile-less defaults for new workspaces. max-cx
++
+TIP: Advanced users can compose and use their own devfiles.
+//add links on "compose" or "devfiles" to relevant sections when those are drafted. max-cx
+* Optional: Private repositories that are hosted on GitHub, GitLab, or Bitbucket require
+.. an authorized application link to be created on the repository-hosting website and
+.. a Kubernetes secret to be applied by the administrator of your organization's {prod-short} instance.
+//two inline links to be added into this par in the next, separate PR. max-cx
+//* "Kubernetes" rather than {platforms-name} requested by Mario. max-cx
+
+.Procedure
+
+. Optional: visit your {prod-short} dashboard page to authenticate as a {prod-short} user to your organization's instance of {prod-short}.
+
+. Visit the URL to start a new workspace using the basic syntax:
+[source,subs="+quotes,+attributes"]
++
+----
+{prod-url}#__<git_repository_url>__
+----
++
+[TIP]
+====
+You can extend this URL with optional parameters:
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?__<optional_parameters>__ <1>
+----
+<1> See xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].
+====
++
+.URL syntax for starting a new workspace that contains a clone of a GitHub-hosted repository
+====
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/tree/__<branch_name>__]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/pull/__<pull_request_id>__]` starts a new workspace with a clone of the branch of the pull request.
+====
++
+After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page in *Progress*.
++
+When the new workspace is ready, the workspace GUI loads in the browser tab.
++
+A clone of the Git repository is present in the filesystem of the new workspace.
++
+NOTE: Each workspace has a unique URL: `{prod-url}#workspace__<unique_url>__`.
+
+[TIP]
+====
+To add a URL as a bookmark in a browser:
```

Comment:
I think the purpose of this tip is to suggest that users bookmark their URLs, not really to explain *how* one creates bookmarks, a commonly understood task.

Reply:
@rolfedh, :D believe it or not, I indeed mean to tell a developer audience how to create a bookmark! The problem here is adding a bookmark for a URL that is not in the address bar of the browser. If I paste one of those URLs in the address bar and hit Enter, the dynamic URL will change, and it will be a completely different URL as a result. In other words, if I want to bookmark a URL without entering it in the address bar, there is a complicated way to do this, and hence this tip.

Reply:
@rolfedh, what do you think? 
![image](https://user-images.githubusercontent.com/78564304/160717029-50f43bab-641f-45ed-a644-3455894da468.png)

Reply:
How about something like this:
```
. Optional: Create a bookmark for the dynamic URL in the <source where they're getting it>, not in the address bar.
+
NOTE: This special procedure is necessary because pasting the dynamic URL into a browser address bar completely modifies the URL contents. This modified URL is less useful than the original dynamic URL.
.. In Mozilla Firefox...
.. In Google Chrome... 
```

Reply:
How about something like this:
```suggestion
. Optional: Create a bookmark for the dynamic URL in the <source where they're getting it>, not in the address bar.
+
NOTE: This special procedure is necessary because pasting the dynamic URL into a browser address bar completely modifies the URL contents. This modified URL is less useful than the original dynamic URL.
.. In Mozilla Firefox...
.. In Google Chrome... 
```